.github/workflows/hoss-test-suite.yml

```diff
@@ -0,0 +1,97 @@
+# This workflow runs HOSS regression test suite, using the tag from the workflow call.
+#
+# Required settings in the repository:
+# Environments (exactly the following strings):
+# - prod
+# - uat
+# - sit
+#
+# Secrets in each environment:
+# - EDL_USER
+# - EDL_PASSWORD
+#
+# Note: A workflow_call event can only have inputs of type boolean, number or
+# string.
+name: Run HOSS test suite
+
+on:
+  workflow_call:
+    inputs:
+      harmony-environment:
+        required: true
+        type: string
+      docker-image-tag:
+        required: false
+        type: string
+        default: 'latest'
+
+  repository_dispatch:
+    types: [hoss]
```

Comment:
Should the types include "all" for each suite? (E.g., for a full Harmony deployment)